### PR TITLE
Embed sandbox

### PR DESCRIPTION
**Pull request by Noojuno**
Currently in the api folder and only supports twitter and itch (and soundcloud once I've got the regex done).

### CODE REVIEW

**Comment by mikekasprzak** [comment]
Just as a reminder, I'm waiting on the non-php version before I accept (since it can all be done in a .html file).

**Comment by Noojuno** [comment]
This is now non-php (although the js is in a php file). Supports these embeds atm: 
screencapture-ludumdare-org-events-testumdare-1-131-embed-test-2018-06-01-22_42_13

File: public-api/vx/embed.php [hunk 1]
```diff
@@ -0,0 +1,248 @@
...
+				new RegExp("([^?=&]+)(=([^&]*))?", "g"),
+				function ($0, $1, $2, $3) {
+					objURL[$1] = $3;
+				}
+			);
+			return objURL;
+		};
+
+		function getEmbedType(url_to_parse) {
+			var twitter_regex = /twitter\.com\/(\w+)\/status(?:es)*\/(\d+)$/;
```
[the remark this answers is not in the document]
Comment (mikekasprzak) [reply public-api/vx/embed.php]: Ooh, good catch.

**Comment by mikekasprzak** [comment]
Just noticed #91. Not blocking, but related.

**Pull request closed without merging**